File: runtimes/swift-5.5/Sources/Runtime/RuntimeOutput.swift
import Foundation

class RuntimeOutput {
    let body: String
    let statusCode: Int
    let headers: [String: String]

    init(body: String, statusCode: Int, headers: [String: String]) {
        self.body = body
        self.statusCode = statusCode
        self.headers = headers
    }
}

File: runtimes/dotnet-7.0/src/RuntimeOutput.cs
using System.Collections.Generic; 

namespace DotNetRuntime
{
	public class RuntimeOutput
	{
		public object Body { get; set; }
		public int StatusCode { get; set; }
		public Dictionary<string, string> Headers { get; set; }

		public RuntimeOutput(object body, int statusCode, Dictionary<string, string> headers)
		{
			Body = body;
			StatusCode = statusCode;
			Headers = headers;
		}
	}
}

